src/extension.ts

```diff
@@ -1278,16 +1320,42 @@ async function createClusterKubernetes(request? : WizardUIRequest) {
     }
 }
 
-async function useContextKubernetes(explorerNode: explorer.ResourceNode) {
+async function useContextKubernetes(explorerNode: explorer.KubernetesObject) {
     const targetContext = explorerNode.metadata.context;
     const shellResult = await kubectl.invokeAsync(`config use-context ${targetContext}`);
     if (shellResult.code === 0) {
-        vscode.commands.executeCommand("extension.vsKubernetesRefreshExplorer");
+        return vscode.commands.executeCommand("extension.vsKubernetesRefreshExplorer");
```

Comment:
The containing method doesn't return a value, so a `return <expr>;` statement feels a bit odd.  Perhaps `refreshExplorer(); return;` instead?